BUILD.md

```diff
@@ -14,10 +14,14 @@ In order to build a binary for `aleph-node` using docker we first need to instal
 distribution, by executing `sudo apt install docker.io` (please consult your distribution's manual describing docker
 installation procedure). Build procedure can be invoked by running:
 ```
-sudo docker build -t aleph-node/build -f nix/Dockerfile.build .
-sudo docker run -ti --volume=$(pwd):/node/build aleph-node/build
+sudo docker build -t aleph-build -f nix/Dockerfile.build .
+sudo docker run -ti --volume=$(pwd):/node/build aleph-build
```

Comment:
I've tried to run it for the second time and got 
```
...
   Compiling finality-aleph v0.5.0 (/tmp/nix-build-aleph-node-unknown.drv-0/aleph-source/finality-aleph)
    Finished release [optimized] target(s) in 2m 52s
installing
[naersk] Using file /tmp/nix-build-aleph-node-unknown.drv-0/tmp.YqVpaGUDB1 to retrieve build (executable) products
[naersk] found executable aleph-node -> /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown/bin/aleph-node
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown
shrinking /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown/bin/aleph-node
strip is /nix/store/p792j5f44l3f0xi7ai5jllwnxqwnka88-binutils-2.31.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown/lib  /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown/bin 
patching script interpreter paths in /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown
checking for references to /tmp/nix-build-aleph-node-unknown.drv-0/ in /nix/store/5kl74gf982qrg9nwxwnm01p4nsvzc8ii-aleph-node-unknown...
error: cannot create symlink '/node/build/result'; already exists
```

Reply:
Added verbose `backup/mv` of the `result` folder.